firmware/arduino/chessboard/chessboard.ino

```diff
@@ -38,8 +38,7 @@ int yMotor[5] = {Y_MOTOR_STEP, Y_MOTOR_DIR, Y_MOTOR_MS1, Y_MOTOR_MS2, Y_AXIS_END
 // ================================
 
 // Distance definitions
-#define MILLIMETERS_PER_SQUARE 63
-#define MILLIMETERS_PER_SPACE 32
+#define MILLIMETERS_PER_UNITSPACE 32
 #define STEPS_PER_MILLIMETER 5
 #define HOME_CALIBRATION_OFFSET 100
 float stepsPerSpace;
```

Comment:
Only one of these is used and the name should have unit in it, like the preprocessor.

Reply:
You can use ctrl+f to figure out which name only shows up once (which is during initialization.